### ISSUE TITLE
Unable to import the project using TFVC

### ISSUE THREAD

**Issue by niiiksh** [microsoft/azure-devops-intellij#67]
I'm trying to import my project to Team Services TFVC but the following error appears: "Error:An argument error occurred: The workspace could not be determined from any argument paths or the current working directory."
Steps:
1. From Android Studio with installed VSTS plugin using TFVC only (no Git installed) create new project
2. VCS - Enable Version Control Integration - TFVC
3. Checkout from Version Control - Team Services TFVC - Select repo and Create Workspace
4. Update Project - OK
Next time when I try to "Update Project" again it shows an "Error:KEY_TF_BAD_EXIT_CODE" 
On event log there are following errors:
"5:53 PM	Plugin Error: Visual Studio Team Services threw an uncaught NotAuthorizedException. Disable Plugin
5:53 PM	NotAuthorizedException: HTTP 401 Unauthorized"
I don't know any other way how to import my project to TFVC
![322342](https://cloud.githubusercontent.com/assets/16052532/24106305/85e0cc3e-0d98-11e7-9cf7-7b7129b21cf6.png)

**Comment by jpricketMSFT** [comment]
@niiiksh - Can you provide some additional information? When you say "Update Project", do you mean the TFVC Update menu item that does a TF Get command? Can you share your intellij log file? If you don't want to share it here, you can send it to me at jpricket <at> microsoft.com.
Thanks,
Jason

**Comment by niiiksh** [comment]
"Update Project" is an option on the VCS menu in Android Studio. 
There is all my logs when I tried to use TFVC on VSTS plugin.


[idea.log.zip](https://github.com/Microsoft/vso-intellij/files/855950/idea.log.zip)

**Comment by jpricketMSFT** [comment]
@niiiksh - the log indicates that the command line cannot determine the workspace. Our plugin uses the root folder of the Android Studio/IntelliJ project to call the command line and get the workspace information. Can you try opening a command prompt, navigating to the root folder and running "c:\tee-clc-14.114.0\tf.cmd workfold". If that doesn't provide the workspace name and collection url, you should try "c:\tee-clc-14.114.0\tf.cmd workspaces /collection:<collectionUrl>" to get a list of the workspaces on that machine. 
If those commands are working, we will have to figure out why Android Studio isn't providing the correct information to the command line.
Thanks,
Jason

**Comment by niiiksh** [comment]
@jpricketMSFT it works now. I've checked out project without files from TFVC and it asked me to create new project, I clicked "yes" and then just copied my created project to the new project folder. Now I'm able to check in files to the version control. But this experience wasn't good. This plugin should import current project to TFVC in one click and ignore files that contains project settings for a specific machine.
